test/Directory.Build.props

```diff
@@ -0,0 +1,5 @@
+<Project>
+  <PropertyGroup>
+    <TestProject>true</TestProject>
```

Comment:
This will mark all projects in the test subfolder as being TestProjects (TestProject == true)

Reply:
seems like this keeps the changes to a minimum